zerver/models.py

```diff
@@ -1309,6 +1309,12 @@ def get_context_for_message(message: Message) -> Sequence[Message]:
         pub_date__gt=message.pub_date - timedelta(minutes=15),
     ).order_by('-id')[:10]
 
+def topic_exists(topic_name: str, recipient: Recipient) -> bool:
+    topic_exists = Message.objects.filter(
+        subject=topic_name,
```

Comment:
This logic is wrong; topics are always matched in a case-insensitive fashion.

So we should do `subject__icontains=` here.

Reply:
Fixed!